release-pipeline.yml

```diff
@@ -1927,6 +1927,9 @@ stages:
                 ./dockerinstall.sh
                 ./buildcontainer.sh DockerfileArm64 ubuntu-arm64
                 ./publishcontainer.sh $(AZCOPY_DOCKER_REG_USER)  $(AZCOPY_DOCKER_REG_PWD) ubuntu-arm64
+
```

Comment:
I've noticed the release pipeline has a lot of copy pasted steps like this, would it be possible to clean some of this up a bit and move some of it to pipeline templates?